include/spdlog/details/registry.h

```diff
@@ -42,8 +42,10 @@ class SPDLOG_API registry {
     // another.
     logger *get_default_raw();
 
-    // set default logger.
+    // set default logger and add it to the registry if not registered already.
     // default logger is stored in default_logger_ (for faster retrieval) and in the loggers_ map.
+    // Note: Make sure to unregister it when no longer needed or before calling again with a new
+    // logger.
```

Comment:
This is a change in the contract, unregister becomes the client's responsibility

Reply:
Only applies to the default logger, which we only set once, so it's fine.